### ISSUE TITLE
Theme is not persistent

### ISSUE THREAD

**Issue by a7med-mahmoud** [withfig/fig#724]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have typed 'git ' and included a screenshot of the Debugger (click the Fig icon in the macOS menu bar)

### Debugger:

<img width="472" alt="Screen Shot 2021-12-17 at 6 36 54 AM" src="https://user-images.githubusercontent.com/68241710/146489899-1452c560-5e36-4559-b24a-fd10160cc8b7.png">

### Description:

I have the theme set to nightowl and it works fine but suddenly after some time the theme changes to the default dark theme

### Environment

```bash
Fig Version: Version 1.0.53 (B339) [Dev] [ABC] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /usr/local/bin/fig
Accessibility: true
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path: <none>
iTerm Integration: application is not present.
Hyper Integration: application is not present.
VSCode Integration: installed!
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: 
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /bin/zsh (4939) - /dev/ttys005
Current window identifier: 15007/% (com.apple.Terminal)
Path: /Users/a7medev/.fig/bin
Fig environment variables:
  - TERM_SESSION_ID=95C780B1-AD8F-445D-B41F-C287525EF330
  - FIG_INTEGRATION_VERSION=5
  - FIG_TERM=1
  - FIG_TERM_VERSION=3
  - FIG_ENV_VAR=1
  - FIG_CHECKED_PROMPTS=1
```

**Comment by SeparateRecords** [comment]
`autocomplete.fontSize` doesn't stick either - the value doesn't change but it seems that it isn't read on startup

**Comment by a7med-mahmoud** [comment]
Guys, please fix this issue. it now switches to the light theme and I get blinded

**Comment by fedeci** [comment]
It has been fixed recently, are you using the last version?

**Comment by a7med-mahmoud** [comment]
```
ig Version: Version 1.0.53 (B339) [Dev] [ABC] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /usr/local/bin/fig
Accessibility: true
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path: <none>
iTerm Integration: application is not present.
Hyper Integration: application is not present.
VSCode Integration: installed!
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: 
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /bin/zsh (52713) - /dev/ttys006
Current working directory: /Users/a7medev/Documents
Current window identifier: 1909/% (com.microsoft.VSCode)
Path: /Users/a7medev/.fig/bin
Fig environment variables:
  - TERM_SESSION_ID=E3E99978-94A7-4056-9D0E-5FE3E5CE0C2E
  - FIG_INTEGRATION_VERSION=5
  - FIG_TERM=1
  - FIG_TERM_VERSION=3
  - FIG_ENV_VAR=1
  - FIG_CHECKED_PROMPTS=1
```

**Comment by a7med-mahmoud** [comment]
I think this is fixed now in Fig 1.0.54, I'll close it. Thanks for the whole Fig team!
